e2e-cypress/files/support/e2e.ts

```diff
@@ -8,8 +8,8 @@ beforeEach(function() {
 })
 
 afterEach(function() {
-  const testName = this.currentTest.fullTitle().replace(/ /g, '_');
-  const fileName = `system-output-${testName}.txt`;
+  const testName = this.currentTest?.fullTitle().replace(/ /g, '_') || 'unknown';
+  const fileName = `system-output-${String(testName)}.txt`;
```

Comment:
I guess the wrapping `String()` is not needed anymore and you can revert the change to:
```suggestion
  const fileName = `system-output-${testName}.txt`;
```